browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -0,0 +1,288 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+	This class is used for managing the accessibility keys of notebookbar control.
+*/
+
+/* global app NotebookbarAccessibilityDefinitions */
+
+var NotebookbarAccessibility = function() {
+
+	this.initialized = false;
+
+	this.activeTabPointers = {
+		id: '',
+		contentList: [],
+		infoBoxList: []
+	},
+
+	this.definitions = new NotebookbarAccessibilityDefinitions();
+	this.tabInfoList = null; // This is to be fetched from NotebookbarAccessibilityDefinitions class at initialization.
+	this.combination = null;
+	this.filteredItem = null;
+
+	this.addInfoBox = function(acnhorElement) {
+		var infoBox = document.createElement('div');
+		infoBox.classList.add('accessibility-info-box');
+		infoBox.textContent = acnhorElement.accessKey;
+		var rectangle = acnhorElement.getBoundingClientRect();
+		infoBox.style.top = (rectangle.top + 20) + 'px';
+		infoBox.style.left = rectangle.left + 'px';
+		document.body.appendChild(infoBox);
+		return infoBox;
+	};
+
+	this.removeCurrentTabInfoBoxesAndAcceleratorKeys = function() {
+		for (var i = 0; i < this.activeTabPointers.contentList.length; i++) {
+			document.getElementById(this.activeTabPointers.contentList[i].id).accessKey = null;
+		}
+
+		for (i = this.activeTabPointers.infoBoxList.length - 1; i > -1; i--) {
+			if (this.activeTabPointers.infoBoxList[i].parentNode !== null)
+				document.body.removeChild(this.activeTabPointers.infoBoxList[i]);
+		}
+	};
+
+	this.addCurrentTabInfoBoxesAndAcceleratorKeys = function(id) {
+		this.activeTabPointers.id = id;
+		this.activeTabPointers.contentList = this.tabInfoList[id].contentList;
+		this.activeTabPointers.infoBoxList = [];
+
+		for (var i = 0; i < this.activeTabPointers.contentList.length; i++) {
+			var element = document.getElementById(this.activeTabPointers.contentList[i].id);
+			if (element) {
+				element.accessKey = this.activeTabPointers.contentList[i].combination;
+				this.activeTabPointers.infoBoxList.push(this.addInfoBox(element));
+			}
```

Comment:
warn here in console so we will know that something was changed (id) and we will be able to fix that